### PR TITLE
fix(PolicyCreate): RHICOMPL-1654 updated empty state for OS major

### PR DESCRIPTION
**Pull request by skateman**
Adjusting the first step of the wizard according the mockup.

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js [hunk 1]
```diff
@@ -86,11 +86,11 @@ const CreateSCAPPolicy = ({ change, selectedBenchmarkId }) => {
...
         <React.Fragment>
             <TextContent>
-                <Text component={TextVariants.h1}>
+                <Text component={TextVariants.h1} className="pf-u-mb-md">
                     Create SCAP policy
                 </Text>
-                <Text component={TextVariants.h4}>
-                    Select the operating system and policy type
+                <Text className="pf-u-mb-md">
+                    Select the operating system and policy type for this policy
```
Comment (vkrizan) [inline src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js:93]: Nit: Missing full stop. Similar within the Policy Type selection.

File: src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js [hunk 2]
```diff
@@ -86,11 +86,11 @@ const CreateSCAPPolicy = ({ change, selectedBenchmarkId }) => {
...
                 </Text>
             </TextContent>
             <Form>
```
Comment (vkrizan) [inline src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js:96]: Can you please also drop the `<br/>` right under the "Operating system"?

**Review by vkrizan — APPROVED** [review]
Works well. Thanks.

File: src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js (hunk 1 above)
Comment (skateman) [reply src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js:93]: What's a full stop :confused:

Comment (vkrizan) [reply src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js:93]: Dot by the end of sentence. :slightly_smiling_face:

Comment (akofink) [reply src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js:94]: Agreed, UX looks for the period/full stop during SPUR.

**Pull request merged**